pkg/aggregator/apiserver/plugin/admission/mutation.go

```diff
@@ -0,0 +1,43 @@
+package admission
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/mattbaird/jsonpatch"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func FromMutationResponse(current []byte, r *backend.MutationResponse) (*admissionv1.AdmissionReview, error) {
+	res := admissionv1.AdmissionResponse{
+		Allowed: r.Allowed,
+		Result: &metav1.Status{
+			Status:  r.Result.Status,
+			Message: r.Result.Message,
+			Reason:  metav1.StatusReason(r.Result.Reason),
+			Code:    r.Result.Code,
+		},
+		Warnings: r.Warnings,
+	}
+
+	patch, err := jsonpatch.CreatePatch(current, r.ObjectBytes)
```

Comment:
@ryantxu i made an assumption here that the plugin would return the mutated object and not a RFC 6902 patch, so i added this to generate the patch. thoughts?

Reply:
I think that is a fair assumption.  But if patch type is part of the k8s contract, should we add it to the grpc/plugin contract?

Reply:
only if we want the plugin to be responsible for calculating the patch. right now it's not needed because the patch is calculated here